main.cpp

```diff
@@ -1,2 +1,5 @@
 int main() {
+  int a;
+  int b;
```

Comment:
warning: variable 'b' is not initialized [cppcoreguidelines-init-variables]

```suggestion
  int b = 0;
```